man/systemd.service.xml

```diff
@@ -1088,6 +1088,279 @@ ExecStart=/bin/echo $ONE $TWO $THREE</programlisting>
   </refsect1>
 
   <refsect1>
+    <title>Service exit codes</title>
+
+    <para>The following are the basic service exit codes defined by the
```

Comment:
There should be some intro, what codes are we talking about and why. E.g. "systemd uses the following codes to signify success or failure when starting processes. It uses both service codes as defined by LSB (link), and defines some values of its own." Then this paragraph.

Another issue: https://github.com/systemd/systemd/commit/70dd455c8e97ba60ab9f8983a4b98372c9ec615e added more verbose explanations. This table and that list or error messages should be coordinated to use the same terminology (i.e. adjust one or the other, whatever makes more sense for each error code).